arcade/examples/isometric_with_billboards.py

```diff
@@ -0,0 +1,167 @@
+# flake8: noqa
+"""
+Using the Orthographic Projector to make a cool isometric scene.
+
+By using a SpriteList and a BillboardList we can create the floor and a little character
+to move around the scene.
+
+The traditional isometric perspective comes from two effects. Firstly the camera rotates "upwards" by 30 degrees.
+Then it is rotated by 45 degrees. Together with an orthographic camera allows you
+to show all 3 dimensions, but depth and vertical height line up so you do loose some dimensionality.
+
+Mathematically this is done using cos and sin of 45 and 60 degrees. We use 60 because it is 90 - 30
+to get the "upwards" motion. These values can be represented by fractions so we can avoid using
+the actual acos and asin methods.
+
+first the 30 degree rotation forward vector
+(0.0, 0.0, -1.0) -> (0.0, sin(60), -cos(60)) = (0.0, sqrt(3)/2, -0.5)
+second the 45 degree rotation around the y-axis
+(0.0, sqrt(3)/2, -0.5) -> (cos(45) * sqrt(3)/2, sin(45) * sqrt(3) / 2.0, -0.5)
+
+this makes the final forward vector
+(sqrt(2) * sqrt(3) / 4, sqrt(2) * sqrt(3) / 4, -0.5) = ((3.0 / 8.0)**0.5, (3.0 / 8.0)**0.5, -0.5)
+
+The up vector is similarly calculated.
+
+If Python and Arcade are installed, this example can be run from the command line with:
+python -m arcade.examples.isometric_with_billboards
```

Comment:
Remember to update this line to match any top-level file rename.
```suggestion
python -m arcade.examples.billboardlist_isometric
```